cocos/scripting/js-bindings/manual/spine/jsb_cocos2dx_spine_manual.cpp

```diff
@@ -262,7 +294,24 @@ jsval spslot_to_jsval(JSContext* cx, spSlot& v)
     if (!tmp) return JSVAL_NULL;
 
     JS::RootedValue jsbone(cx, spbone_to_jsval(cx, *v.bone));
-    JS::RootedValue jsattachment(cx, spattachment_to_jsval(cx, *v.attachment));
+    
+    jsval jstemp = JSVAL_NULL;
+    if (v.attachment->type == spAttachmentType::SP_ATTACHMENT_REGION) {
+        jstemp = spregionattachment_to_jsval(cx, *((spRegionAttachment*)(v.attachment)));
+    }
+    else if (v.attachment->type == spAttachmentType::SP_ATTACHMENT_MESH ||
+             v.attachment->type == spAttachmentType::SP_ATTACHMENT_LINKED_MESH) {
+        jstemp = spmeshattachment_to_jsval(cx, *((spMeshAttachment*)(v.attachment)));
+    }
+    else if (v.attachment->type == spAttachmentType::SP_ATTACHMENT_BOUNDING_BOX){
+        
+        jstemp = spboundingboxattachment_to_jsval(cx, *((spBoundingBoxAttachment*)(v.attachment)));
+    }
+    else {
+        jstemp = spattachment_to_jsval(cx, *(v.attachment));
+    }
+    JS::RootedValue jsattachment(cx, jstemp);
+    
```

Comment:
The added logic (except BoundingBoxAttachment related) has been added in branch v1.4.
So could you please send the PR to branch v1.4? 

Sorry for the inconvenience!